### ISSUE TITLE
Inconsistent default value for zend.assertions

### ISSUE THREAD

**Issue by mterwill** [iusrepo/php72u#15]
[PHP docs](https://www.php.net/manual/en/function.assert.php#function.assert.expectations) specify the default value for `zend.assertions` as `1`. However, this package declares it as `-1`, optimizing away `assert` statements and any code within them.

https://github.com/iusrepo/php72u/blob/954856caec675b1d6676dfac281b49fe7c0cbe87/php.ini#L1478

e.g. if you have

```php
assert($foo->someOperation());
```

with `zend.assertions = -1`, `$foo->someOperation()` will never be called.

**Comment by carlwgeorge** [comment]
That's not a choice we made, that was inherited from the [Fedora spec file](https://src.fedoraproject.org/rpms/php/blob/f29/f/php.ini#_1478) that we forked from.  IUS can diverge from Fedora when it makes sense, but I don't see a lot of value in that here.  The PHP docs you linked specifically say that -1 is "production mode", so I don't know why the package would use anything else.  That doesn't stop someone from changing the value on their own system.

If you disagree and would like to see this changed, I suggest [opening a bug for the Fedora package](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora&component=php).  If Fedora implements that change, I'll backport it to the IUS packages.
